Innowise.Clinic.Auth.Extensions/ConfigurationManager.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Text;
+using Innowise.Clinic.Auth.Jwt;
+using Innowise.Clinic.Auth.Jwt.Interfaces;
+using Innowise.Clinic.Auth.Mail;
+using Innowise.Clinic.Auth.Mail.Interfaces;
+using Innowise.Clinic.Auth.Persistence;
+using Innowise.Clinic.Auth.Validators.Custom;
+using Innowise.Clinic.Auth.Validators.Identity;
+using Innowise.Clinic.Auth.Validators.Interfaces;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Innowise.Clinic.Auth.Extensions;
+
+public static class ConfigurationManager
+{
+    public static IServiceCollection ConfigureIdentity(this IServiceCollection services)
+    {
+        services.AddIdentity<IdentityUser<Guid>, IdentityRole<Guid>>(options =>
+            {
+                options.Password.RequireDigit = false;
+                options.Password.RequiredLength = 6;
+                options.Password.RequireLowercase = false;
+                options.Password.RequireNonAlphanumeric = false;
+                options.Password.RequireUppercase = false;
+            })
+            .AddEntityFrameworkStores<ClinicAuthDbContext>()
+            .AddDefaultTokenProviders()
+            .AddPasswordValidators();
+
+
+        return services;
+    }
+
+    private static IdentityBuilder AddPasswordValidators(this IdentityBuilder builder)
+    {
+        builder.AddPasswordValidator<MaximalPasswordLengthValidator<IdentityUser<Guid>>>();
+
+        return builder;
+    }
+
+    public static IServiceCollection ConfigureEmailServices(this IServiceCollection services)
+    {
+        services.AddScoped<IEmailHandler, EmailSender>();
+        return services;
+    }
+
+    public static IServiceCollection ConfigureJwtAuthentication(this IServiceCollection services,
+        IConfigurationSection jwtData)
```

Comment:
jwtData looks like it is token itself so rename it ito smth like `jwtConfig `or `jwtSettings`